tests/basics/tstring_basic1.py

```diff
@@ -0,0 +1,143 @@
+try:
+    from string.templatelib import Template, Interpolation
+except ImportError:
+    print("SKIP")
+    raise SystemExit
+
+try:
+    exec('t"test"')
+except SyntaxError:
+    print("SKIP")
+    raise SystemExit
+
+print("=== Parser error tests ===")
+try:
+    exec('t_empty = t"{}"')
+    print(f"Empty expr: {t_empty.interpolations[0].value}")
+except SyntaxError as e:
+    print(f"Empty expr: SyntaxError - {e}")
+
+# Whitespace in expression (MicroPython preserves, CPython strips trailing)
+t_ws = t"{ 42 }"
+print(f"Whitespace: {str(t_ws)}")
```

Comment:
This behaves differently from CPython, but keeping it unchanged for now since CPython may have a bug.
Currently checking with Core developers to confirm if it's a bug.